src/control/lib/control/mocks.go

```diff
@@ -595,6 +594,15 @@ type (
 	}
 )
 
+// temp copy from common/test to avoid polluting lib/control with test deps
```

Comment:
Could the solution for these be as simple as renaming them to mocks_test.go?

Reply:
No, because there are tests outside of this package that import them (e.g. dmg, etc.). I have experimented with a couple of different solutions (e.g. creating a new `package/test/mocks.go` convention for the whole codebase), but I wanted to keep this PR focused on the `common` -> `common/test` split in order to make it more manageable.